src/app/functions/server/getUserBreaches.ts

```diff
@@ -81,3 +87,67 @@ export async function getUserBreaches({
     phoneBreaches,
   };
 }
+
+/**
+ * NOTE: new function to replace getUserBreaches
+ *
+ * @param user
+ */
+export async function getSubscriberBreaches(
+  user: Session["user"]
+): Promise<SubscriberBreach[]> {
+  const subscriber = await getSubscriberByEmail(user.email);
+  const allBreaches = await getBreaches();
+  const breachesData = await getSubBreaches(subscriber, allBreaches);
+  return breachesData;
+}
+
+interface GuideExperienceBreaches {
```

Comment:
nit: should this be "Guide" or "Guided"?
```js
export function guidedExperienceBreaches(...)
const guidedExperienceBreaches: GuideExperienceBreaches = { ... }
```

Reply:
We use `guided experience`

Reply:
So shouldn't this be:

```diff
-GuideExperienceBreaches
+GuidedExperienceBreaches
```

Reply:
Ah, didn't see that one. Thanks for the catch!